bundler.go

```diff
@@ -104,6 +104,10 @@ type ConfigurationBind struct {
 	// The package of the generated file
 	// Defaults to "main"
 	Package string `json:"package"`
+
+	// The path used for the LD Flags
+	// Defaults to the `Package` value
+	PackagePath string `json:"package_path"`
```

Comment:
I'd rather not put this option in the `ConfigurationBind` struct. I'd rather put it in the `Configuration` struct and name the attribute `LDFlagsPackage`

Reply:
I agree 100%, love the location and name. I only added it here because `Package` was being used for this purpose, so I just tried to fit it in here